cmd/check_container.go

```diff
@@ -168,17 +170,40 @@ func getContainerPolicyExceptions(ctx context.Context, pc pyxisClient) (policy.P
 
 func checkContainerPositionalArgs(cmd *cobra.Command, args []string) error {
 	if len(args) != 1 {
-		return fmt.Errorf("a container image positional argument is required")
+		return fmt.Errorf("a container image positional argument is required or a flag is not specified as --flag=value")
 	}
 
+	cmd.Flags().VisitAll(func(f *pflag.Flag) {
+		if f.Changed && strings.HasPrefix(f.Value.String(), "--submit") {
```

Comment:
Any reason not to use `strings.Contains`?

Reply:
I suppose that would work... Any reason not to use strings.HasPrefix? It seems the scenario of --submit showing up anywhere other than the beginning seems slim to none.